### PR TITLE
added linters and README

### PR DESCRIPTION
**Pull request by banzaman**
In this milestone, I added

- Created linters.yml file 
- Deleted linter.yml 
- Made corrections
- Updated liners
- Editted README
- Deleted test. md

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @banzaman, 

**This is a great project you have going on, well-done lad**. 
   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️

- [ ] Add a PR description explaining all the implemented features you have added to this project.
- [ ] It is a standard practice to always capitalize your commit messages.

_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
```
Comment (mke2111) [inline .github/linters.yml:19]: - [ ] The linters are not detected by Github. 
- Add the linters.yml file inside the workflows directory which will also be inside the `.github` directory.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```
Comment (mke2111) [inline README.md:33]: - [ ] Delete all the options/steps that are not needed for this project.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+👤 **MARK RUBANZA**
+
+- GitHub: [@githubhandle](https://github.com/banzaman)
+- Twitter: [@twitterhandle](https://twitter.com/banzamarq10)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (mke2111) [inline README.md:46]: - [ ] The issues link is broken. Add the correct URL to your issues page.

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```
Comment (mke2111) [inline .gitignore:2]: - [ ] The test.md file still exists in your repo. Add the test.md file to this ignore file so that its ignored by git

**Review by karvel-code — CHANGES_REQUESTED** [review]
Hi @banzaman , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Please ensure that you add a PR description stating the changes that you have made with regards to the project.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by barackm — APPROVED** [review]
Hi @banzaman, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**